CollAction/Views/Shared/_Layout.cshtml

```diff
@@ -49,5 +49,8 @@
     <environment names="Staging,Production">
         <script src="~/js/bundle.js"></script>
     </environment>
-    </body>
+
+    @RenderSection("scripts", required: false)
```

Comment:
I added the same fix in my branch, depending on whose branch is merged first, someone will have to undo their fix.